pyevdi/Makefile

```diff
@@ -11,7 +11,8 @@ LIBABI := 1
 
 INCL_DIRS := -I../library $$(python3 -m pybind11 --includes)
 
-CXXFLAGS := $(INCL_DIRS) -std=c++11 -fPIC $(CXXFLAGS)
+# -fvisibility=hidden https://pybind11.readthedocs.io/en/stable/faq.html#someclass-declared-with-greater-visibility-than-the-type-of-its-field-someclass-member-wattributes
+CXXFLAGS := $(INCL_DIRS) -std=c++20 -fPIC -fvisibility=hidden $(CXXFLAGS)
```

Comment:
Why you decided to up the minimal C++ version to `c++20`, is that necessary? I don't see you do any change that requires newer C++ standard

Reply:
Ah, `span` requries. Well, that's to the point of having separate commits for every functional change, because it's hard to find relevant stuff 😊